.env

```diff
@@ -0,0 +1,2 @@
+STATIC_URL="http://localhost:3000/"
+PORT=3000
```

Comment:
can we make this variable more explicit about what it's for? see example https://github.com/wildone/synology-docker/blob/main/.env